{{cookiecutter.role_name}}/.gitlab-ci.yml

```diff
@@ -21,23 +21,23 @@ Ansible lint:
   after_script:
     - rm -rf ~/ansible-lint-rules
   tags:
-    - delegated
+    - aws
 
 Docker {{ cookiecutter.role_name }}:
   stage: deployment test
   script:
     - molecule test -s default
   tags:
-    - docker
+    - aws
 
 AWS {{ cookiecutter.role_name }}:
   variables:
-    EPC_REGION: AWS-EUCENTRAL
+    AWS_REGION: us-east-1
```

Comment:
should be replaced for using variables value from cookiecutter

Reply:
Done